docs/integrations/saas-cloud/sentinelone.md

```diff
@@ -0,0 +1,395 @@
+---
+id: sentinelone
+title: Sumo Logic App for SentinelOne
+sidebar_label: SentinelOne
+description: The Sumo Logic App for SentinelOne offers functionality for monitoring and analyzing your organization's SentinelOne Threats and Agents logs.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/sentinelone-icon.png')} alt="sentinelone-icon.png" width="100" />
+
+The SentinelOne App for Sumo Logic is designed to provide security professionals with a comprehensive view of their organization's security posture. Based on Sumo Logic’s SentinelOne Source, this app allows you to quickly ingest data from your SentinelOne agents into Sumo Logic for real-time analysis.
+
+With the SentinelOne App, you can gain valuable insights into your endpoint security data, including threat intelligence, endpoint activity, and system health. You can monitor your endpoints for suspicious behavior and quickly respond to any security incidents that may arise. 
+
+The app provides pre-built dashboards that enable you to quickly visualize your SentinelOne data and gain insights into your organization's security posture. You can customize these dashboards to meet your specific needs and monitor the metrics that matter most. With the SentinelOne App for Sumo Logic, you can:
+* Ingest data from SentinelOne agents into Sumo Logic for real-time analysis and response.
+* Monitor your endpoints for suspicious behavior and respond to security incidents in real-time
+* Gain valuable insights into your organization's security posture with pre-built dashboards
+* Customize dashboards to meet your specific needs and monitor the metrics that matter most to you
+
+In summary, the SentinelOne App for Sumo Logic provides security professionals with the tools they need to monitor and respond to security incidents in real-time, ensuring the protection of their organization's critical assets.
+
+## Log Types
+
+The Sumo Logic App for SentinelOne consumes Threats and Agents logs, refer to the [Threats](https://usea1-partners.sentinelone.net/api-doc/api-details?category=threats&api=get-threats) and [Agents](https://usea1-partners.sentinelone.net/api-doc/api-details?category=agents&api=get-agents) documentation.
+
+### Sample Log Messages
```

Comment:
It'd be great if we could write both log messages under the Sample Log Messages heading and queries under the Sample Queries section.

Reply:
Agreed! @priyansh-sumo , I've just fixed this